### ISSUE TITLE
assimulo.solvers.sundials.CVodeError: 'The right-hand side function failed at the first call. At time 0.000000.'

### ISSUE THREAD

**Issue by jschueller** [modelon-community/PyFMI#184]
the simulation of the following model exported in a ME mode FMU file:
```
model FloodModel
  parameter Real Hd = 3 "height of the dyke";
  parameter Real Zb = 55.5 "altitude of the river banks";
  parameter Real L = 5000 "river length";
  parameter Real B = 300 "river width";
  
  Real Zc;
  Real Zd;
  Real H(start=1);
  Real alpha;

  input Real Q;
  input Real Ks;
  input Real Zv;
  input Real Zm;
  output Real S;

equation
  S = Zc-Zd;
  Zc = H+Zv;
  Zd = Zb+Hd;
  alpha = (Zm-Zv)/L;
  H = if Ks > 0 and alpha > 0 then (Q/(Ks*B*sqrt(alpha)))^0.6 else 1;

end FloodModel;
```

... with the following code:
```
import pyfmi

path_fmu = "/tmp/FloodModel.fmu"
model = pyfmi.load_fmu(path_fmu)
result = model.simulate()
print(result["S"][-1])
```

... fails with:
```
Traceback (most recent call last):
  File "/home/schueller/projects/openturns/otfmi/test/test_pyfmi.py", line 5, in <module>
    result = model.simulate()
             ^^^^^^^^^^^^^^^^
  File "src/pyfmi/fmi.pyx", line 8221, in pyfmi.fmi.FMUModelME2.simulate
  File "src/pyfmi/fmi.pyx", line 378, in pyfmi.fmi.ModelBase._exec_simulate_algorithm
  File "src/pyfmi/fmi.pyx", line 374, in pyfmi.fmi.ModelBase._exec_simulate_algorithm
  File "/usr/lib/python3.11/site-packages/pyfmi/fmi_algorithm_drivers.py", line 800, in solve
    self.simulator.simulate(self.final_time, self.ncp)
  File "assimulo/ode.pyx", line 191, in assimulo.ode.ODE.simulate
  File "assimulo/ode.pyx", line 314, in assimulo.ode.ODE.simulate
  File "assimulo/explicit_ode.pyx", line 135, in assimulo.explicit_ode.Explicit_ODE._simulate
  File "assimulo/explicit_ode.pyx", line 221, in assimulo.explicit_ode.Explicit_ODE._simulate
  File "assimulo/solvers/sundials.pyx", line 2063, in assimulo.solvers.sundials.CVode.integrate
  File "assimulo/solvers/sundials.pyx", line 2096, in assimulo.solvers.sundials.CVode.integrate
assimulo.solvers.sundials.CVodeError: 'The right-hand side function failed at the first call. At time 0.000000.'
```

if I export the model in a CS mode FMU it goes fine:
```
$ python test/test_pyfmi.py 
Simulation interval    : 0.0 - 1.0 seconds.
Elapsed simulation time: 0.010959399998682784 seconds.
-57.5
```

Any clue what might be wrong ? The error message is not very explicit.

I'm using latest pyfmi 2.10.3, assimulo 3.4.1
I tried old and new versions of sundials without luck

**Comment by PeterMeisrimelModelon** [comment]
Resolved with #187 in 2.10.4.

**Comment by jschueller** [comment]
thanks a lot
